src/InterruptAccelStepper.h

```diff
@@ -64,7 +64,15 @@ template <typename STEPPER> class InterruptAccelStepper
     {
         LOG(DEBUG_STEPPERS, "[IAS-%d] setSpeed(%f)", STEPPER::TIMER_ID, speed);
         _speed = speed;
-        STEPPER::moveTo(_speed, INT32_MAX);
+
+        if (speed == 0.0f)
```

Comment:
Shouldn't compare`==`  with floats

Reply:
yeah i know, but to be honest i hoped that 0.0f should always be correct because sign, exponent, mantissa are zero for this special value. and thus should be comparable without issues. what do you think?

Reply:
In theory we could use epsilon comparison here ... but not sure which epsilon value would be ok here.

Reply:
Good idea, will do that. Thank you!

Reply:
done